### PR TITLE
Surprise: Merry Xmas, revised tweakdata.

### PR DESCRIPTION
**Pull request by DiscendoVox**
Many of these are adjustments to formatting to ease future revision.
Others are increases in resistance, health, damage, or falloff in response to calls for greater challenge. 
Some may need to be tweaked back down based on feedback, but the game should be much less easy to solo with these.

### CODE REVIEW

File: lua/lib/tweak_data/charactertweakdata.lua [hunk 1]
```diff
@@ -2017,14 +1757,15 @@ function CharacterTweakData:_init_deathvox(presets)
...
+	self.deathvox_heavyar.HEALTH_INIT = 101 -- new with final 2017 pass.
 	self.deathvox_heavyar.headshot_dmg_mul = 3
+	self.deathvox_heavyar.damage.explosion_damage_mul = 0.9
 	self.deathvox_heavyar.access = "swat"
 	table.insert(self._enemy_list, "deathvox_heavyar")
 	
 	self.deathvox_lightshot = deep_clone(self.city_swat)
 	self.deathvox_lightshot.detection = presets.detection.normal
-	self.deathvox_lightshot.suppression = presets.suppression.hard_agg -- tentative.
+	self.deathvox_lightshot.suppression = presets.suppression.no_supress -- tentative.
```
Comment (DiscendoVox) [inline lua/lib/tweak_data/charactertweakdata.lua]: This edit was in error, light shotgunners should keep hard_agg suppression.

File: lua/lib/tweak_data/charactertweakdata.lua [hunk 2]
```diff
@@ -2092,6 +1836,7 @@ function CharacterTweakData:_init_deathvox(presets)
 	self.deathvox_medic = deep_clone(self.medic)
 	self.deathvox_medic.tags = {"medic"} --just making sure tag applies.
 	self.deathvox_medic.detection = presets.detection.normal
+	self.deathvox_heavyar.damage.hurt_severity = presets.hurt_severities.no_hurts -- added to make code consistent.
```
Comment (DiscendoVox) [inline lua/lib/tweak_data/charactertweakdata.lua]: this, too, was in partial error, and should be only_fire_and_poison_hurts

File: lua/lib/tweak_data/charactertweakdata.lua [hunk 3]
```diff
@@ -2173,35 +1920,37 @@ function CharacterTweakData:_init_deathvox(presets)
 
 	self.deathvox_sniper.use_factory = true -- Use a factory weapon
 	self.deathvox_sniper.factory_weapon_id = {"wpn_deathvox_sniper"}
-	self.deathvox_sniper.HEALTH_INIT = 15
```
Comment (DiscendoVox) [inline lua/lib/tweak_data/charactertweakdata.lua]: Crap, this should stay at 150.

File: lua/lib/tweak_data/charactertweakdata.lua [hunk 4]
```diff
@@ -2173,35 +1920,37 @@ function CharacterTweakData:_init_deathvox(presets)
...
+	self.deathvox_sniper.HEALTH_INIT = 34 -- new with final 2017 pass.
 	self.deathvox_sniper.headshot_dmg_mul = 3
 	self.deathvox_sniper.is_special_unit = "sniper"
 	self.deathvox_sniper.access = "sniper"
 
 	self.deathvox_sniper_assault = deep_clone(self.deathvox_sniper)
 	self.deathvox_sniper_assault.move_speed = presets.move_speed.very_fast
-	self.deathvox_sniper_assault.deathguard = true --tentative. This was apparently a big problem in RAID, but that unit may be implemented differently.
+	self.deathvox_sniper_assault.deathguard = true
 	self.deathvox_sniper_assault.HEALTH_INIT = 15
```
Comment (DiscendoVox) [inline lua/lib/tweak_data/charactertweakdata.lua]: This, instead, should be 340.

File: lua/lib/tweak_data/charactertweakdata.lua (hunk 3 above)
Comment (Iamgoofball) [reply lua/lib/tweak_data/charactertweakdata.lua]: its fine, see below

**Pull request merged**